debug.log

```diff
@@ -0,0 +1 @@
+[0811/195304.174:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Probably we want to remove this log from the repo and add this file to .gitignore

Reply:
Removed and updated .gitignore to prevent it from being inadvertently re-added in the future.